generated/niscope/niscope.proto

```diff
@@ -465,8 +465,8 @@ enum ExportableSignals {
   EXPORTABLE_SIGNALS_NISCOPE_VAL_READY_FOR_ADVANCE_EVENT = 6;
   EXPORTABLE_SIGNALS_NISCOPE_VAL_READY_FOR_START_EVENT = 7;
   EXPORTABLE_SIGNALS_NISCOPE_VAL_READY_FOR_REF_EVENT = 10;
-  EXPORTABLE_SIGNALS_NISCOPE_VAL_REF_CLOCK = 100;
   EXPORTABLE_SIGNALS_NISCOPE_VAL_5V_OUT = 13;
+  EXPORTABLE_SIGNALS_NISCOPE_VAL_REF_CLOCK = 100;
```

Comment:
Note to self: I keep seeing this ref_clock one jump around. Guessing it's something goofy in hapigen metadata causing this with every update. Will look into.

Reply:
I'm guessing it's not explicitly sorted when it should be? This time it appears they happen to be sorted numerically. I'm going to mark it resolved and see if we can fix that up separately.

Reply:
Yeah.. it's re-arranged the same way in enums.py so I'm guessing it's something in the hapigen metadata (probably something to do with 'position' key on that end)